src/components/RegistrationForm.tsx

```diff
@@ -354,6 +360,15 @@ export default function RegistrationForm({ handles, wardens, className }) {
                   <small>This field is required</small>
                 </p>
               )}
+              {hasValidationErrors &&
+                state.username &&
+                isDangerousHandle(state.username) && (
```

Comment:
I'm not sure this will update when user changes the value in the input. I think we need to track this with useState the same way discordUsername is validated.

Reply:
Oh I see, It's resolved now :)